mmv1/templates/terraform/pre_update/network_security_address_group_operation.go.erb

```diff
@@ -0,0 +1 @@
+project = billingProject
```

Comment:
Actually, this line is before getting `billingProject` in update and delete method in the generated file and isn't working as expected
https://github.com/modular-magician/terraform-provider-google-beta/blob/be49b211522d9e2b34b84ff66268f8a9f8be5bf0/google-beta/services/networksecurity/resource_network_security_address_group.go

Is it safe to move the pre_update just from [line 776](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/resource.erb#L776) to [line 794](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/resource.erb#L794) before the line of sendRequest in template file resource.erb?

Reply:
Good catch! Let's make these `""` in that case. Can you file an issue to fix the inconsistency in where `pre_` methods are? That should happen in a dedicated PR.

Reply:
Sure. I will file an issue.